Doxyfile

```diff
@@ -1413,7 +1413,7 @@ CHM_FILE               =
 HHC_LOCATION           =
 
 # The GENERATE_CHI flag controls if a separate .chi index file is generated
-# (YES) or that it should be included in the master .chm file (NO).
+# (YES) or that it should be included in the main .chm file (NO).
```

Comment:
These are template comments by Doxygen, I'm not sure what references will break if we change those.